### PR TITLE
Feat/delete io repo

### PR DESCRIPTION
**Pull request by felix-lambert**
We will not work on IOS, only android, so no use to have the ios native repo

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+/**
+ * @format
+ */
+
+import {AppRegistry} from 'react-native';
```
Comment (lutangar) [inline index.js:5]: I saw `eslint` as a dependency so you might borrow the rules of the extension codebase.
...this give me to think that we also could create a repository for eslint configuration and share it across projects.
Yet another point to discuss when we'll speak about splitting the apps @achauve @JalilArfaoui

**Review by julienadamcom — APPROVED** [review]
Ok no problem, iOS is not is the scope (at least right now)

File: index.js (hunk 1 above)
Comment (lutangar) [reply index.js:5]: I mentioned `eslint` but I should also mention, prettier, the idea is of course to let the machine do the formatting and to format this file with a consistent code style.

**Review by lutangar — APPROVED** [review]

File: index.js (hunk 1 above)
Comment (felix-lambert) [reply index.js:5]: Ok I will apply eslint extension for mobile. I'm not sure getting very well what you mean with prettier? Do you think we should remove it or .gitignore it?

Comment (JalilArfaoui) [reply index.js:5]: Also, using TypeScript instead of FlowType would keep us closer to `extension`

**Pull request merged**